src/Component.php

```diff
@@ -34,13 +34,13 @@ private function moveFolder(string $dataFolderName): void
         foreach ($finder as $file) {
             $fs->mkdir($this->getDataDir() . "/out/" . $dataFolderName . "/" .  $file->getRelativePath());
             $fs->rename($file->getPathname(), $this->getDataDir() . "/out/" . $dataFolderName . "/" . $file->getRelativePathname());
```

Comment:
A co ještě vyextrahovat tady `$this->getDataDir() . "/out/" . $dataFolderName . "/"`?

Reply:
Lezou mi řádky ven z Githubu :)

Reply:
```
$fs->rename($file->getPathname(), $this->getDataDir() . "/out/" . $dataFolderName . "/" . $file->getRelativePathname());
```
vs
```
$destination = $this->getDataDir() . "/out/" . $dataFolderName . "/" . $file->getRelativePathname();
$fs->rename($file->getPathname(), $destination);
```
ušetří jen pár znaků. nebo bys to ještě víc atomizoval?

```
$destinationPath = $this->getDataDir() . "/out/" . $dataFolderName . "/";
$fs->rename($file->getPathname(), $destinationPath . $file->getRelativePathname());
```
to bych radši udělal
```
$fs->rename(
    $file->getPathname(), 
    $this->getDataDir() . "/out/" . $dataFolderName . "/" . $file->getRelativePathname()
);
```

Reply:
:+1: